todo_app/SeleniumTest/Tests/test_e2e.py

```diff
@@ -1,3 +1,4 @@
+""" 
```

Comment:
If we're disabling tests, we should either:
* Mark them to be skipped with `@pytest.mark.skip(reason="these need updating")` so that future users can see what the reason is and reason when they should be switched on
* Or delete them entirely!

Reply:
skipped integration tests